modules/javafx.graphics/src/main/java/javafx/scene/AmbientLight.java

```diff
@@ -26,15 +26,16 @@
 package javafx.scene;
 
 import com.sun.javafx.scene.AmbientLightHelper;
-import com.sun.javafx.scene.DirtyBits;
-import com.sun.javafx.scene.NodeHelper;
 import com.sun.javafx.sg.prism.NGAmbientLight;
 import com.sun.javafx.sg.prism.NGNode;
 import javafx.scene.paint.Color;
 
 /**
- * Defines an ambient light source object. Ambient light is a light source
- * that seems to come from all directions.
+ * A light that illuminates an object from all directions equally.
+ * <p>
+ * {@code AmbientLight}s can represent strong light sources in an enclosed area where the lights bounces from many
+ * objects, causing them to be illuminated from many directions. A strong light in a room and moonlight are common light
```

Comment:
What examples would you use?

If a light in a room is strong it will barely cast shadows since its reflections from all directions eliminate them.

Maybe I should mention that Ambient light can be used with a dark color to provide a base weak lighting of the environment, and on top of it use other lights.

Reply:
I think the description should focus on the meaning of the respective term in the lighting equation, and not on a non-technical analogy. In this case, the analogy is a bit misleading on several aspects, including the fact that ambient lighting is not dependent on an area being enclosed. Here's a suggestion:
```
Ambient lights add a constant term to the amount of light reflected by each point on
the surface of an object, thereby increasing the brightness of the object uniformly and
independently of the orientation of its surfaces. It is often used to represent the base
amount of illumination in a scene.
```

Reply:
I think the key aspects of an Ambient light are that: the light seems to come from all directions (and thus has no position or direction), and that it illuminates objects independently of the position or orientation of the object.

I don't have a problem with also giving a real-world analogy if one can be found that makes it easier to understand without causing confusion.

Reply:
I tried to unify some of the points that were brought up. See if you would like more changes.